### PR TITLE
Dev to main 

### PR DESCRIPTION
**Pull request by MichaelLELU**

### CODE REVIEW

File: client/src/pages/DetailsPage.jsx
Comment (Pierrejuppin) [inline client/src/pages/DetailsPage.jsx:1]: the file should not be deleted

Comment (elhayanich) [inline client/src/pages/DetailsPage.jsx:1]: Component needed, it should not be deleted

File: client/src/pages/DetailsPage.module.css
Comment (elhayanich) [inline client/src/pages/DetailsPage.module.css:1]: this is the style sheet needed for the DetailsPage component, it should not be deleted as well

**Review by Pierrejuppin — COMMENTED** [review]
The files DetailsPage and DetailsPage.module should not be deleted

**Review by ioayoub — APPROVED** [review]

**Review by HippolyteWeber — APPROVED** [review]

**Review by elhayanich — APPROVED** [review]

**Review by Pierrejuppin — APPROVED** [review]

**Pull request merged**